### ISSUE TITLE
Restart packet update?

### ISSUE THREAD

**Issue by Zexeta** [OpenKore/openkore#2643]
Packet update please?

**Comment by Zexeta** [comment]
:( drop/exp event is active on restart atm, can someone please update or post a tutorial. Last time I spent hours and no one would explain where the additional packets restart uses came from.

**Comment by Zexeta** [comment]
They apparently didn't even bother to update them this week.

**Comment by Fadreus** [comment]
Congratulation on discovering that ! :clap: :clap: :clap: 
It must be hard for you to spell each word one by one and finally, 3 days later, you succeed in reading !

Achievement Lv 1 unlocked - Reading English Text.

**Comment by Nipodemos** [comment]
You can do your own update if not patient.
the script of allanon is open source and you can use to generate the update

But you don't want to make this, do you? You just want to complain

**Comment by Zexeta** [comment]
My comment was directed at the restart admins didn't update so new ones weren't needed. They didn't even patch the exe. Not at you guys.

I did try doing it myself. People directed me to a script I used and it didn't work. It covered the packets used in other servers. I tried to point out multiple times that iro uses extra packets that the script was not extracting all of them.

Someone else finally posted saying the same so they checked, posted the new packets n said oh sorry most servers don't do use these additional packets. This was of course after a certain someone spammed emotes n made insulting comments like a child the entire time then had nothing to say when I was right. 

No one really explained how to get the extra packets after that.

**Comment by Fadreus** [comment]
> restart admins

There is no such thing as admin.
People doing this as hobby.

> iro uses extra packets that the script was not extracting all of them.

People using allanon tool for iRO, not PEEK.

> someone spammed emotes n made insulting comments like a child the entire time then had nothing to say when I was right.

I spam emotes coz nothing I say is going your dumb head.
Someone already post the steps using allanon tool and most people already succeed.
The decreasing comment about iRO after he posted the steps proved it.

> No one really explained how to get the extra packets after that.

People explain a lot, it just you lazy to search.
You want people to spoon feed you.

If there is no answer to your solution, I won't mocking you.
The only reason I'm mocking someone is because everything usually just in front of their eyes or just few clicks away and they  are too blind to see it or just too lazy to use their brain.

I'm not talking about computing stuff, basically just simple reading stuff.
Like how you ignore to use latest version before making this issue.
If you read, why won't you follow the rules when reporting?
Where the report templates?
Why you ignoring report template and things to know?
Are you so special that rules don't apply to you?

I don't want to promote laziness & spoon-feeder.
I want this community to grow by being able to search for solution without depending too much on others because since the last decade as you can see in forums, those being feed are unlikely can contribute back because they never learn, they just waiting for people.

But then again, people like you will often give more drama to what I said which is why I keep my "nasty" comment at minimum and this place not for drama.
If you put some effort reading past issues, merely every issue here is about implementing updates and fixing openkore, NOT PLACE TO TEACH PEOPLE.

If you want to ask about PEEK, go comment at PEEK repo.
If you want to ask about allanon tool, go comment at his gist.
If you want to ask something but don't know where, go IRC.
If that don't work, use forum under discussion.

Either way, all of this written in front of this repo but you never read, so you never know.

**Comment by Fadreus** [comment]
> restart admins

What's the point of saying that here?
Why don't you post something to your cat here too?
OR can I say something to my grandma here?
What you think this place is?

> People directed me to a script I used and it didn't work. It covered the packets used in other servers.

That's because you are asking wrong question. You asked about shuffle without being specific on what server, of course he explain about the general use of it. As if he know what you mean. We here don't have ability to read mind and which is why i keep telling BE FUCKING SPECIFIC WHEN REPORTING. The report template exist for a reason.

> iro uses extra packets that the script was not extracting all of them.

People using allanon tool for iRO, not PEEK.

> someone spammed emotes n made insulting comments like a child the entire time then had nothing to say when I was right.

I spam emotes coz nothing I say is going your dumb head.
Someone already post the steps using allanon tool and most people already succeed.
The decreasing comment about iRO after he posted the steps proved it.

> No one really explained how to get the extra packets after that.

People explain a lot, it just you lazy to search.
You want people to spoon feed you.

If there is no answer to your solution, I won't mocking you.
The only reason I'm mocking someone is because everything usually just in front of their eyes or just few clicks away and they  are too blind to see it or just too lazy to use their brain.

I'm not talking about computing stuff, basically just simple reading stuff.
Like how you ignore to use latest version before making this issue.
If you read, why won't you follow the rules when reporting?
Where the report templates?
Why you ignoring report template and things to know?
Are you so special that rules don't apply to you?

I don't want to promote laziness & spoon-feeder.
I want this community to grow by being able to search for solution without depending too much on others because since the last decade as you can see in forums, those being feed are unlikely can contribute back because they never learn, they just waiting for people.

But then again, people like you will often give more drama to what I said which is why I keep my "nasty" comment at minimum and this place not for drama.
If you put some effort reading past issues, merely every issue here is about implementing updates and fixing openkore, NOT PLACE TO TEACH PEOPLE.

If you want to ask about PEEK, go comment at PEEK repo.
If you want to ask about allanon tool, go comment at his gist.
If you want to ask something but don't know where, go IRC.
If that don't work, use forum under discussion.

Either way, all of this written in front of this repo but you never read, so you never know.

**Comment by Fadreus** [comment]
> My comment was directed at the restart admins didn't update so new ones weren't needed. They didn't even patch the exe. Not at you guys.

![image](https://user-images.githubusercontent.com/32188494/53315238-27f8aa00-38fd-11e9-8768-52dd52aef7ca.png)

> then had nothing to say when I was right.

Because you are still wrong and I'm speechless of what I could do to make you brain cells connected.
You are asking about PEEK but using allanon tool is more simple. You went this wrong because someone asked about PEEK for his educational purpose, not for iRO. IDK why you jump on his ship and think iRO update using PEEK every week.

> No one really explained how to get the extra packets after that.

Really? Then what is this? Love letter? Or some message directed to his neighbor's cat like you just did? 
![image](https://user-images.githubusercontent.com/32188494/53315533-a99d0780-38fe-11e9-834f-b3d57bbcf8b3.png)


I can still prove you are wrong because you don't do enough reading.
Like how you ignore report template when start this issue and using latest version before making new issue? But then again, in human history, arguing with a ignorant will never bear any result, which is why I keep silent and emoting on every ignorant comments.

[![](http://img.youtube.com/vi/biX1Qyd7oEU/0.jpg)](http://www.youtube.com/watch?v=biX1Qyd7oEU "")

**Comment by Nipodemos** [comment]
Ok sorry for misunderstanding
about the script for updates and additional packages and stuff, this is not my area, Unfortunaly I don't know anything about it

**Comment by Fadreus** [comment]
> nothing to say when I was right.

Yeah, you were right. You never wrong.
In fact, you are so right that no one can go against you word, My King.
Yes, allanon script is not working. Peek is not working.

All hail Zexeta, King Of Right.
He is so right that everything is not working.
In fact, those which is wrong working properly.

When the rules said to use latest version before starting this issue; OR use report template.
### NO. THIS IS WRONG.
### AS A KING I DON'T CHECK. IF THERE IS NO PR, IT MUST BE NOT WORKING & FUCK WITH TEMPLATE. I'M KING. I DO AS I WISH. #2643 #2425 #2409 #1020

When all issue must be about openkore:
### NO, I UPDATE MY WINDOWS AND ITS NOT WORKING. SO IT MUST BE OPENKORE FAULT. #2360 
### I'M A KING OF RIGHT. MY WORD IS RULE.

PSS: I don't reply last time coz sometimes, when someone so wrong at so many level and so fucked up his brain, there is no saving. Plus, I don't like drama in github but in forum it's oke. :rofl: 

Admins: Feel free to remove this comment if it was inappropriate, I don't mind if he call me clown like last time but I can't let go of which is not right. He is wrong and people (him) must know that. Otherwise, many would follow his footsteps; nag nag nag & nag.

**Comment by Fadreus** [comment]
> Someone else finally posted saying the same so they checked, posted the new packets n said oh sorry most servers don't do use these additional packets. This was of course after a certain someone spammed emotes n made insulting comments like a child the entire time then had nothing to say when I was right.

Nope, you still wrong.
- 1st, you talking about PEEK because 1 guy asked for his educational purpose, not for iRO & you thought iRO update using PEEK. **So you were wrong here.**
- 2nd, you were whining about PEEK for iRO while iRO use allanon tool. **Wrong again.**
- 3rd, you said the replacing the value wont work & try to said that many times but the problem is missing handler & not about your shuffle, PEEK or allanon tool., **So you wrong, AGAIN.**
- 4th, you said I had nothing to say because you were right. While it's true I'm speechless of your "smartness" but **you were wrong** about you being right.
- 5th, You said allanon packet didn't work because you didn't follow Gudoh instruction carefully when running the script, **so you wrong again** saying it's not working.
- 6th, before starting this issue, the rules said to use latest version. **Is this right to ignore rules? You tell me, right or wrong?**
- 7th, You think success is earned by very tiny amount of effort like only few hours trying and very little times you've tried. Life said **you are dead wrong again.**

Now, enlighten me.
You can do cross reference to your statement which is you claim to be true.
I'm very happy to correct you.

But what do I gain from this? Nothing.
For you? Might be able to solve your problem if you put aside your ego & put some effort on it.

Anyway, if you insist that you are still right. Fine. If people wanna believe the earth is flat, so be it. No one easily change people belief. If I can, I would be Messiah already LMAO.
